cassandra-diagnostics-commons/src/main/java/io/smartcat/cassandra/diagnostics/utils/Utils.java

```diff
@@ -48,4 +50,22 @@ public static void setHostname(final String hostname) {
         Utils.hostname = hostname;
     }
 
+    /**
+     * Get system name.
+     *
+     * @return system name
+     */
+    public static String getSystemname() {
```

Comment:
`systemName` is a configuration options (with default value). Configuration options that are not module- or reporter-specific are expected to be accessed via the `Configuration` class. That's why I believe it should not be part of `Utils`.